Makefile

```diff
@@ -26,13 +26,13 @@ ifeq (x86_64, $(TARGET))
   DASM_FLAGS = -D X64=1
 endif
 ifeq (x86, $(TARGET))
-  CC= gcc -m32
-  CFLAGS += -DIR_TARGET_X86
+  CFLAGS += -m32 -DIR_TARGET_X86
   DASM_ARCH  = x86
   DASM_FLAGS =
 endif
 ifeq (aarch64, $(TARGET))
-  CC= aarch64-linux-gnu-gcc --sysroot=$(HOME)/php/ARM64
+  # FIXME: This overwrites clang from CI
+  #CC = aarch64-linux-gnu-gcc --sysroot=$(HOME)/php/ARM64
```

Comment:
I use cross compilation. It's much faster then QEMU.  This is not a blocker, just info. I may continue using cross-compilation.

Reply:
Ah! So none of the generated code is executed at the moment, just compared? It might still make sense to keep using QEMU if that ever becomes the case, if it's not much slower. I'll try to gather some numbers.